deployment/Dockerfile

```diff
@@ -1,12 +1,12 @@
 FROM python:slim
 
 WORKDIR /zambeze
-COPY requirements.txt requirements.txt
-RUN pip3 install -r requirements.txt
 COPY . . 
-RUN pip3 install .
+RUN pip install -e .
```

Comment:
This change was just to ease development only. The `-e` helps development because I don't need reinstall everything for any change I do in a file. For regular use of Zambeze, we only need the typical `pip install .`